Library/Homebrew/formula_specialties.rb

```diff
@@ -36,7 +42,6 @@ def standard_instructions home_name, home_value=libexec
        * On Bash, add them to `~/.bash_profile`.
        * On Zsh, add them to `~/.zprofile` instead.
 
-      export JAVA_HOME="$(/usr/libexec/java_home)"
       export AWS_ACCESS_KEY="<Your AWS Access ID>"
       export AWS_SECRET_KEY="<Your AWS Secret Key>"
       export #{home_name}="#{home_value}"
```

Comment:
Maybe we should put this(e.g. `export EC2_AMITOOL_HOME="#{libexec}"`) into `write_env_script`. So users don't need to set them manually.